backend/src/starknet.ts

```diff
@@ -0,0 +1,20 @@
+import { StarknetIdNavigator } from "starknetid.js";
+import { RpcProvider, constants } from "starknet";
+
+export const getStarknetId = async (
+  address: string
+): Promise<string | null> => {
+  const provider = new RpcProvider({
+    nodeUrl: "your_mainnet_node_url",
```

Comment:
Use our node URL from Scarb.toml – `http://34.22.208.73:6060/v0_7`

Or better, load it from an environment variable

Reply:
Okay